apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/brevbaker/BrevbakerService.kt

```diff
@@ -36,7 +36,7 @@ class BrevbakerService(
             with(redigerbarTekstRequest) {
                 BrevbakerRequest.fra(
                     brevkode(brevKodeMapper, generellBrevData),
-                    brevDataMapper.brevData(this),
+                    brevDataMapperRedigerbartUtfall.brevData(this),
```

Comment:
Synes kanskje vi burde ha tilsvarende kode i denne servicen for ferdigstilling? Nå opprettes den requesten et annet sted såvidt jeg kan skjønne? Hadde vært enklere å se gangen i det hvis begge disse variantene var ganske like.

Reply:
Liker tanken! Kan gi det eit forsøk og sjå kva eg får til 🤞 Men blir i ny PR, denne er stor nok 😅

Reply:
Prøvde meg litt, og innser at eg er usikker på om eg trur det er best at ferdigstilling blir meir som redigerbar eller at redigerbar blir meir som ferdigstilling. Så vi får fortsette å forenkle litt meir, så blir det kanskje tydelegare etter kvart.